src/Transformers/JsonApi/params/serializeSorting.js

```diff
@@ -9,6 +9,6 @@ export default function serializeSorting(parameterBag) {
   });
 
   return {
-    sort: sorting.join(','),
+    sort: sorting.join(',') || {},
```

Comment:
Wellicht gewoon null/leeg {} returnen ipv een {sort: {}} als er geen sortering is?